.github/workflows/clj-build.yml

```diff
@@ -0,0 +1,142 @@
+name: Clojure Build
+on:
+  workflow_call:
+    inputs:
+      runner:
+        required: false
+        type: string
+        default: self-hosted
+      target_branch:
+        required: false
+        type: string
+        default: main
+jobs:
+  build:
+    name: Build App
+    runs-on: ${{ inputs.runner }}
+    steps:
+      - name: cancel prior runs
+        uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
This version is using Node.js 12 actions which are deprecated. I think the latest version of this action did update to Node.js 16 but it also warns towards using the native _concurrency_ property to cancel workflows (https://github.com/styfle/cancel-workflow-action) - your call.

Reply:
Added the concurrency approach and then triggered two workflows in quick succession for sfmc-sink. The first one was canceled in favor of the latter as expected
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/7834668/231493503-94d7181f-e577-47b5-986b-00e55f624cd1.png">